### DISCUSSION TITLE
tutorials/growing-graphql-api-dgs-lab/06-wrapping-up

### DISCUSSION THREAD

**Discussion by giscus** [apollographql-education/content-feedback#120] [course feedback]
# tutorials/growing-graphql-api-dgs-lab/06-wrapping-up

https://www.apollographql.com/tutorials/growing-graphql-api-dgs-lab/06-wrapping-up

**Comment by mkatsman** [comment]
PLease put input id outside of a check box. It is impossible to copy the string.

**Reply by mkatsman** [reply]
I see, it is already done.
